src/Comsave/SafeSalesforceSaverBundle/Producer/RpcSfSaverClient.php

```diff
@@ -7,49 +7,55 @@
 use OldSound\RabbitMqBundle\RabbitMq\RpcClient;
 use PhpAmqpLib\Exception\AMQPTimeoutException;
 
-/**
- * Class RpcSfSaverClient
- * @package Comsave\SafeSalesforceSaverBundle\Producer
- */
 class RpcSfSaverClient
 {
+    public const REQUEST_EXPIRATION = 50;
```

Comment:
Although you just refactored this to a constant, isn't this value is extremely low.

["The value of the expiration field describes the TTL period in milliseconds."](https://www.rabbitmq.com/ttl.html#per-message-ttl-in-publishers)

Reply:
In the Rabbit bundle it's in seconds.
https://github.com/php-amqplib/RabbitMqBundle#rpc-or-replyresponse